VideoApi/VideoApi.DAL/Queries/GetConferencesTodayForStaffMemberByHearingVenueNameQuery.cs

```diff
@@ -0,0 +1,46 @@
+using Castle.Core.Internal;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+
+namespace VideoApi.DAL.Queries
+{
+    public class GetConferencesTodayForStaffMemberByHearingVenueNameQuery : IQuery
+    {
+        public IEnumerable<string> HearingVenueNames { get; set; }
+    }
+
+    public class GetConferencesTodayForStaffMemberByHearingVenueNameQueryHandler : IQueryHandler<GetConferencesTodayForStaffMemberByHearingVenueNameQuery, List<Conference>>
+    {
+        private readonly VideoApiDbContext _context;
+
+        public GetConferencesTodayForStaffMemberByHearingVenueNameQueryHandler(VideoApiDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<Conference>> Handle(GetConferencesTodayForStaffMemberByHearingVenueNameQuery query)
+        {
+            var today = DateTime.Today;
+            var tomorrow = DateTime.Today.AddDays(1);
+
+            return await _context.Conferences
+                .Include(x => x.Participants)
+                .Include(x => x.Endpoints)
+                .Include(x => x.MeetingRoom)
+                .AsNoTracking()
+                .Where(x => x.ScheduledDateTime >= today && x.ScheduledDateTime < tomorrow)
+                .Where(x => x.MeetingRoom != null && x.MeetingRoom.AdminUri != null && x.MeetingRoom.JudgeUri != null
+                            && x.MeetingRoom.ParticipantUri != null && x.MeetingRoom.PexipNode != null)
+                .Where(x => !query.HearingVenueNames.IsNullOrEmpty() && query.HearingVenueNames.Contains(x.HearingVenueName))
```

Comment:
Could we move the first condition to the start of the method and throw something since we expect a venue list.

Reply:
I wonder if we even need thatconsidering hearing venue names gets validated before it ever gets to this query, do you want me to just remove this check perhaps?

Reply:
just tested, all my tests are passing, the view hearing button is disabled if you dont choose a hearing in video web as well, plus even when I inspect element and removed the disabled tag and then press the view hearing button, it still works fine because I have a validation for the request being made to this endpoint